tests/IVIMmodels/unit_tests/test_ivim_fit_linear.py

```diff
@@ -2,6 +2,8 @@
 import numpy.testing as npt
 import pytest
 import torch
```

Comment:
Do we need torch?

Reply:
No, I'll remove it. I think I added it because it seemed like it was being used by other people and I wanted to add it as an option but ended up not doing that.